### PR TITLE
deps: update jquery version

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
I reran the torture tests and they failed again on org.jitsi.meet.test.SharedVideoTest.backToSharedVideoTest. While SharedVideoTest is known to be flaky, it seems this failure might be legit.

**Comment by saghul** [comment]
Doh, I thought this was going to be just a version bump :-(

**Comment by saghul** [comment]
Thanks for looking into it @virtuacoplenny! I'll get back to this when I have some time.

**Comment by virtuacoplenny** [comment]
So the problem is the youtube video thumbnail is being left as display:none when it should be displayin:inline-block. I have no memory of how any of this js visibility works...

**Comment by virtuacoplenny** [comment]
Something about Filmstrip#resizeThumbnails is setting the thumbnail to display:none.

**Comment by virtuacoplenny** [comment]
I have a feeling it has to do with: https://jquery.com/upgrade-guide/3.0/#breaking-change-show-hide-and-toggle-methods-now-respect-more-stylesheet-changes. The thumbnails by default are display:none so it could be jquery is honoring that. Remote video thumbnails might be working because of a call to VideoLayou#remoteVideoActive() which explicitly calls a .show() on the video element. Now I'm not confident of this path but it seems like a decent first lead. A couple workarounds I can think of would be understanding why video starts with display:none, explicitly calling show() on the shared thumbnail, or removing jquery instead of updating it.

**Comment by virtuacoplenny** [comment]
Adding @yanas as a reviewer now that I've become a committer. Based on getJSON also changing and breaking, but torture tests not catching it, I'll have to do a manual sanity check of the various meet features before I have some confidence to merge.

File: modules/UI/shared_video/SharedVideo.js [hunk 1]
```diff
@@ -282,6 +282,7 @@ export default class SharedVideoManager {
 
             thumb.setDisplayName('YouTube');
             VideoLayout.addRemoteVideoContainer(self.url, thumb);
+            VideoLayout.resizeThumbnails(false, true);
```
Comment (yanas) [inline modules/UI/shared_video/SharedVideo.js]: Was this related in any way to the jquery update or was it a separate bug?

Comment (virtuacoplenny) [reply modules/UI/shared_video/SharedVideo.js]: It's a bug with the jquery update that most of the comments in this PR are about.

Comment (virtuacoplenny) [reply modules/UI/shared_video/SharedVideo.js]: More specifically it's the implementation that I settled on to resolve YouTube thumbnails not displaying. More details are in the commit message but in short: I failed and used a workaround.

**Pull request merged**